mixer/adapter/prometheus/prometheus.go

```diff
@@ -107,7 +118,20 @@ func (b *builder) clearState() {
 
 func (b *builder) SetMetricTypes(map[string]*metric.Type) {}
 func (b *builder) SetAdapterConfig(cfg adapter.Config)    { b.cfg = cfg.(*config.Params) }
-func (b *builder) Validate() *adapter.ConfigErrors        { return nil }
+func (b *builder) Validate() (ce *adapter.ConfigErrors) {
+	if b.cfg.MetricsExpirationPolicy != nil {
+		if b.cfg.MetricsExpirationPolicy.MetricsExpiryDuration <= 0 {
+			ce = ce.Appendf("metricsExpiryDuration",
+				"metricsExpiryDuration %v is invalid, must be > 0", b.cfg.MetricsExpirationPolicy.MetricsExpiryDuration)
+		}
+		if b.cfg.MetricsExpirationPolicy.ExpiryCheckIntervalDuration <= 0 {
```

Comment:
perhaps use a reasonable default value? what are the tradeoffs of different values here?